war/src/main/less/modules/table.less

```diff
@@ -44,6 +44,10 @@
         a {
           color: var(--table-header-foreground);
           font-weight: 700;
+
+          span {
+            width: 24px;
```

Comment:
This is weird. Why does that work? Why would every `span` inside an `a` in a `th` need to be this wide? This is the kind of thing that would benefit from a comment.